Jordan_Class.py

```diff
@@ -0,0 +1,202 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri May 22 18:15:13 2020
+
+@author: User
+"""
+import re # importing regular expression operations, which we need later to access the libraries
```

Comment:
I don't know how much you (or our teachers) care about this, but maybe it's at least worth pointing out:

According to the Python style guide, you should leave two spaces between code and within-line comments (and in general keep those types of comments to a minimum).